custom/icds_reports/static/js/directives/cas-export/cas-export.directive.js

```diff
@@ -129,6 +130,24 @@ function CasExportController($window, $location, locationHierarchy, locationsSer
         return locationsService.getLocations(level, locationsCache, vm.selectedLocations, true);
     };
 
+    vm.userHaveAccessToAllLocations = function (locations) {
```

Comment:
it might be clearer to rename this to something like `userHaveAccessToLevel` or `accessToAllLocationsForLevel` or something that makes clear its only one level, and distinguishes it from the very similarly named variable that already exists and is used in the same conditional below

Reply:
done